mps_youtube/main.py

```diff
@@ -3130,7 +3150,7 @@ def play_range(songlist, shuffle=False, repeat=False, override=False):
         if returncode == 42:
             n -= 1
 
-        elif returncode == 43:
+        elif returncode == 0:
             break
```

Comment:
If we break on returncode of 0 only the first item of a selection will be played.